dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -37,6 +37,39 @@ const moveToTestPanel = () => {
   cy.get('h1').contains(TEST_PANEL).should('exist');
   cy.wait(delay);
 };
+let panelCount = 1;
+const panelName='TestPanel';
```

Comment:
Space inconsistent. I mentioned this in earlier reviews to check for all inconsistency. @abasatwar , don't we have linters here, which can tell us about these problems?